Testing/SystemTests/lib/systemtests/systemtesting.py

```diff
@@ -667,6 +671,46 @@ def start(self, script):
         os.remove(tmp_file.name)
         return results
 
+    def start_in_parent_thread(self, script):
```

Comment:
Can we rename this `start_in_current_process`? I think it's a more accurate description of what's happening?

Reply:
Good point, will do